### ISSUE TITLE
Scaled image disappears while full image is loading

### ISSUE THREAD

**Issue by sarbogast** [ideaismobile/IDMPhotoBrowser#111]
Why does the scaled image disappears while the full image is loading? See the following video:
http://quick.as/yD20iL4w

I think the user experience would be much better if we showed the scaled image underneath the activity indicator while the full image is loading.

**Comment by RebornSoul** [comment]
I am also experiencing this issue, absolutely agreed with the opinion that underlying image should be visible while main image is loading.

**Comment by hsavit1** [comment]
yes,  I experience this as well. Sometimes, the image doesnt even load up!

**Comment by GorkaMM** [comment]
Any idea what's causing this behavior? I'd be happy to fix it and make a pull request.
